docs/Erlang Distribution.md

```diff
@@ -0,0 +1,92 @@
+# Erlang Distribution
+
+## Static Configuration
+
+The simplest way to configure Erlang distribution is to statically (i.e., at
+compile time) set the node name and cookie. This is done by editing the
+`rel/vm.args` file, which contains the command-line arguments used to launch
+the Erlang/OTP virtual machine.
+
+You would simply add `-name` and `-setcookie` parameters, as follows:
```

Comment:
My preference here would be to use `-sname` or at least to document both `-sname` and `-name`. `-sname` has the advantage that the node's name can be autogenerated based on the serial number which makes things really easy for how I've been using Erlang distribution. The main problem with `-sname` is that it works best if the DHCP server can register the name with DNS, and it's hit or miss whether people's routers support this. @mobileoverlord: Which do you use now? `-name` or `-sname`? And is it only for debug or have you used Erlang distribution in production?

Regarding use of `nerves.local`, I had thought that that would only work if you had an mDNS service running on your Nerves device. Or is this not true?